New score

EhJackson88 opened the pull request:
- Display score text at bottom of screen
- Score += 10 when fireball destroyed ATM
- Score text size = 15
- Tested and working

EhJackson88 committed:
Added score text to bottom of screen

- Added Canavas / UI text
- Placed in bottom left on game screen
- Created script to display score

EhJackson88 committed:
Score updates

- Changed size of text to 15
- Set scoreText.text = to playerScore
- Updated score in fireballBehavior to add points when fireball has been
destoryed
- Tested scoreUpdate and is working (can play with point values later)

awebeer256 reviewed (changes requested):
-Rename "isAShield" to "IsAShield" (capital I).
-Fix the missing IsAShield script on the shields, and on the shield prefab. (Doing the renaming above might fix this automatically.)
-Fix the missing IsACharacter and IsAPlayerOwned scripts on the player, and on the player prefab.

EhJackson88 committed:
Sch score (#30)

* Fixed missing script on Player

- Added IsACharacter and IsAPlayerOwned script to player

* Changed IsAShield Script

- Changed name of script to NameChange

* Changed NameChange script back to IsAShield

- Changed name for NameChange script back to IsAShield

awebeer256 reviewed (changes requested):
-You still didn't fix missing scripts on the player PREFAB.
-This scene now has two canvases. I'm pretty sure the professor said that that was a very bad thing. Put the score display on the canvas that already existed.
-The score display is a text that isn't on a panel. Fix that.
-You added another GameMgr component to the score display object? I...what? I don't even. Use the one that's on GLOBAL.

EhJackson88 committed:
Score fixes  (#31)

* Fixed missing script on Player

- Added IsACharacter and IsAPlayerOwned script to player

* Changed IsAShield Script

- Changed name of script to NameChange

* Changed NameChange script back to IsAShield

- Changed name for NameChange script back to IsAShield

* Added scripts to player PREFAB

- Added two scripts to player prefab -> IsACharacter, IsAPlayerOwned

* Deleted ScoreCanvas and added panel / text to existing canvas

- Deleted ScoreCanvas and text
- Created new ScorePanel on Canvas
- Created text to display score on ScorePanel

* Score GUI updates

- Made scoreText field public
- Added ScoreText field object to GameMgr script
- Deleted line adding points when fireball is destroyed for testing
purposes

EhJackson88 committed:
Added scoreText field to GameMgr script

- null reference occruing because text object wasn't given in script
- added scoreText to scrip

awebeer256 committed:
made score actually display
